content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,160 @@
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Controller pattern
+
+A controller tracks at least one Kubernetes resource type.
+These [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+and their associated data are the desired state, so the
+controller(s) for that resource are responsible for making the current
+state come closer to that desired state.
+
+The controller might carry the action out itself; more commonly, in Kubernetes,
+a controller will send messages to the
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} that have
+useful side effects.
+
+### Control via API server
+
+The controller for Job is an example of a built-in controller that
+makes all of its changes by interacting with your cluster's API server.
+
+{{< glossary_tooltip term_id="job" >}} is a Kubernetes resource that
+runs a {{< glossary_tooltip term_id="pod" >}}, or perhaps several,
+to carry out a task and then stop.
+
+(Once [scheduled](/docs/concepts/scheduling/), Pod objects become part
+of the desired state for a kubelet).
+
+When the Job controller sees a new task it makes sure that, somewhere
+in your cluster, the kubelets on a set of Nodes are running the right
+number of Pods to get the work done.
+The Job controller does *not* run any Pods or containers
+itself. Instead, the Job controller tells the API server to create or remove
+Pod [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/).
+Other components in the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+act on the new information (there are new Pods to schedule and run),
+and eventually the work is done.
+
+After you create a new Job, the desired state is for that Job to
+be completed. The Job controller makes the current state be nearer
+to the desired state, by creating Pods that do the work you
+wanted.
+
+Controllers also update the objects that configure them
+For example: once the work is done for a Job, the Job controller
+updates that Job object to mark it `Finished`.
+
+(This is a bit like how some thermostats turn a light off to
+indicate that the your room is now at the temperature you set).
+
+### Direct control
+
+By contrast with Job, some controllers need to make changes to
+things outside of your cluster.
+
+For example, if you use a control loop to make sure there
+are enough {{< glossary_tooltip text="Nodes" term_id="node" >}}
+in your cluster, then that controller needs something outside the
+current cluster to set up new Nodes when needed.
+
+This kind of controller finds its desired state from the API
+server and then communicates directly with an external system
+to bring the current state closer in line.
+
+(There actually is a controller that horizontally scales the
+nodes in your cluster. See
+[Cluster autoscaling](https://kubernetes.io/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)).
+
+## Desired vs. current state {#desired-vs-current}
```

Comment:
s/vs./versus

I think "vs." is probably an acceptable exception to [avoiding Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) in English. It's probably less acceptable for localization purposes, so let's spell it out.